omnibus/config/software/datadog-agent.rb

```diff
@@ -236,4 +243,25 @@
   unless windows_target?
     delete "#{install_dir}/uselessfile"
   end
+
+  # TODO: move this to omnibus-ruby::health-check.rb
+  # check that linux binaries contains OpenSSL symbols when building to support FIPS
+  if fips_mode? && linux_target?
+      command "ls -R #{install_dir}"
```

Comment:
This looks like a left over debug :)

Reply:
You're right, I've removed it here 6ff101f6fc03e348a5d4d37babed791a71dcb932